tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -49,6 +52,41 @@ PACKAGE_LIST="autotools-dev \
         rsync libunwind8 libpng16-dev libexpat1-dev \
         python3-setuptools python3-numpy python3-wheel python python3-pip python3-pytest \
         libprotobuf-dev libprotobuf9v5 protobuf-compiler"
+else
+    add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
```

Comment:
add a comment that this else block is for ubuntu18.04 so it's clear for others who might modify this file in the future.